luvit-loader.lua

```diff
@@ -207,7 +207,7 @@ local function loader(dir, path, bundleOnly)
     if package.loaded[fullPath] then
       return package.loaded[fullPath]
     end
-    local module = assert(loadfile(fullPath))()
+    local module = assert(loadfile(fullPath))(fullPath)
```

Comment:
I've never heard of this.  If it's harmless then it's fine, but they seem like different typed of names to me.